### PR TITLE
Task 37 - StateOrderCmd

### PR DESCRIPTION
**Pull request by brianmontero18**

### CODE REVIEW

File: t/state_order_cmd.t [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+my $purchase_cmd = StateOrderCmd->new(@parameters);
+my $output = $purchase_cmd->execute(OrderServiceFake->new());
+is( $output->get_output(), "Pedido: 95332 \nNombre: jjlopez", 'The status of the order has been shown' );
+
+#error case
+my @eparameters = ('50312');
+my $epurchase_cmd = StateOrderCmd->new(@eparameters);
+my $error = $epurchase_cmd->execute(OrderServiceFake->new());
+is( $error->get_output(), "El pedido '$eparameters[0]' no fue encontrado", "Error: the order does not exist");
+
```
Comment (elin3t) [inline t/state_order_cmd.t:21]: @brianmontero18 add done_testing() function so perl know that is finished!

Comment (brianmontero18) [inline t/state_order_cmd.t:21]: done_testing() added!

**Comment by elin3t** [comment]
+1

**Comment by earandap** [comment]
+1

**Pull request merged**